### ISSUE TITLE
Crash on first start of pack

### ISSUE THREAD

**Issue by VallenFrostweaver** [Freebi85/Dungeons-Dragons-Space-Shuttles#113]
DD&SS version: 4.7b
Forge: 14.23.5.2807
Pressed "play" on Twitch launcher after downloading and got this screen at about part way through 3/7 on the load screen:
![crash](https://user-images.githubusercontent.com/13642054/54681119-7f0c2c00-4b03-11e9-8395-06e9a7e2097a.png)

...and this is the link I get when I click: https://paste.dimdev.org/qikeculome.mccrash
I also ran it again with 8GB RAM and 9GB as well but it provided the same results.
8GB (set at 9): https://paste.dimdev.org/exeheyapob.mccrash
9GB (set at 10): https://paste.dimdev.org/osobidemij.mccrash

**Comment by VallenFrostweaver** [comment]
NOTE: Loading this pack up on MultiMC resulted in a similar failure but my wife didn't have an issue. Also tried going back a version to 4.7 and still crashed.

**Comment by Freebi85** [comment]
This is not necessary a pack specific issue. This issue has been around for years, I've seen reports of this being an issue all the way back to 2013. 
It's supposed to have something to do with the graphics card but nobody seems to have found a concrete solution yet. I read that it should help changing graphics card OpenGL settings, expecially anything connected with texture memory.
I can't say if this is a 100% solution.

**Comment by VallenFrostweaver** [comment]
Here's the results: 
-Default settings of OpenGL triple buffering off & Texture Filtering Quality to Standard = crash
-Turn my OpenGL triple buffering settings on = crash
-Set Texture Filtering Quality to High = crash
-Set Texture Filtering Quality to Performance = crash
I would draw a conclusion from this that it is unlikely to be effecting it though it could still have a possibility that this is effecting the issue as well as something else. Do you have any other recommendations? I appreciate your attempt to remedy the issue regardless.

**Comment by cyber01** [comment]
I can be wrong, but maybe just a very weak video card or a driver problem?

**Comment by VallenFrostweaver** [comment]
I currently have:
![temp](https://user-images.githubusercontent.com/13642054/54689590-c1d7ff00-4b17-11e9-9170-5a75be97ae87.png)

**Comment by Freebi85** [comment]
Difficult to say. Your computer doesn't seem to be too bad. Slowly I run out of ideas.
Remove foamfix (rendering manipulating mod) and see if it still happens.

**Comment by cyber01** [comment]
You use any resource/texturepack? What about java? version and 32/64 bit?

**Comment by VallenFrostweaver** [comment]
@cyber01 No resource/texturepack (no Optifine either). Java is up to date. 64 bit.
@Freebi85 I disabled SmoothFont and Foamfix (I also noticed that Dynamic Surroundings was disabled by default). This seems to have allowed me to pass Gandalf after all.

I got in. \o/

Thank you for this as it seems to have resolved the issue of failing to load. :D Much appreciation! I had no idea it would red line so badly though on boot up. Wow that's heavy. Anyway, thanks again!

**Comment by Freebi85** [comment]
Yeey ✌️ 💛 
Still odd lol. 
Should you like the pack you have to remember to do this every time after an update. I'll close the issue. Have fun =)

**Comment by VallenFrostweaver** [comment]
UPDATE (just in case others need the info): After disabling those mods and succeeding in loading the pack I upped the RAM to 9500 from 8500 and it failed to load. I returned the RAM to 8500 and it failed still. I found that I had another instance of Minecraft running on the first try and had shut it down during the successful attempt. I repeated this again, launching the DD&SS pack while I had another pack running in the background (a small custom pack) and it worked. What the heck makes it work like this is beyond me.
